pkg/querier/tenantfederation/merge_queryable.go

```diff
@@ -143,7 +149,8 @@ func (m *mergeQuerier) LabelValues(name string, matchers ...*labels.Matcher) ([]
 		name = m.idLabelName
 	}
 
-	return m.mergeDistinctStringSlice(func(ctx context.Context, q storage.Querier) ([]string, storage.Warnings, error) {
+	return m.mergeDistinctStringSlice(func(ctx context.Context,
```

Comment:
Sorry for being unclear! It's perfectly fine to pass `ctx` to `mergeDistinctStringSlice()`. I just meant to rollback the splitting to 2 lines.

Reply:
done